### PR TITLE
Add a type of 'all' to set all types in one command

### PR DESCRIPTION
**Pull request by gileswells**
Full disclosure, I am unsure if this works. 

The build succeeds but I haven't dug into figure out where the build outputs to / if it actually does what I hope it will do. This came out of a frustration of my audio switching for output but then not for SMS chimes or for my input mic for Zoom calls.

### CODE REVIEW

**Comment by gileswells** [comment]
@deweller please? I'd try to compile it myself for personal use but I have no idea where to begin on that.

**Comment by earthtone0ne** [comment]
Came here looking for this feature! I'll try to compile @gileswells branch (I'd have to learn how to do that, too!) but in the meantime, I added these commands to my `.bashrc`:

```bash
local headset_name="Logitech USB Headset"
local internal_speaker="MacBook Pro Speakers"
local internal_mic="MacBook Pro Microphone"
# switch input and output
alias headset='SwitchAudioSource -s "{headset_name}"; SwitchAudioSource -s "{headset_name}" -t input;'
alias speaker='SwitchAudioSource -s "{internal_speaker}"; SwitchAudioSource -s "{internal_mic}" -t input;'
# switch to specified input/output (e.g. `aud h`) or toggle if not specified 
function aud() {
  if [[ ! -f /usr/local/bin/SwitchAudioSource ]] then
    return 1
  fi
  if [[ -z "$1" ]] then 
    SwitchAudioSource -n
    SwitchAudioSource -n -t input
  elif [[ "$1" == "h" ]] then
    headset
  elif [[ "$1" == "s" ]] then
    speaker
  fi 
}
```

**Comment by gileswells** [comment]
@earthtone0ne yeah I just run all three under the alias of `airpod`. Now that I have Monterey installed I just need to see what keyboard shortcut isn't going to mess with the apps I use and I'll drop that into the Shortcuts app rather than have to drop into terminal.

File: audio_switch.c [hunk 1]
```diff
@@ -112,8 +112,10 @@ int runAudioSwitch(int argc, const char * argv[]) {
 					typeRequested = kAudioTypeInput;
 				} else if (strcmp(optarg, "output") == 0) {
 					typeRequested = kAudioTypeOutput;
-				} else if (strcmp(optarg, "system") == 0) {
-					typeRequested = kAudioTypeSystemOutput;
+                } else if (strcmp(optarg, "system") == 0) {
```
Comment (dferg) [inline audio_switch.c:115]: @gileswells you should probably fix the indention here to match the original code

**Comment by deweller** [comment]
Thanks for confirming that this works.  I'll check this out and merge this week.

**Comment by gileswells** [comment]
> I'll check this out and merge this week.

Thoughts @deweller?

**Comment by reflexdemon** [comment]
I was always wanting the all feature. Please merge this and kindly publish the build

**Pull request merged**